projects/sandbox/vizapp/pyproject.toml

```diff
@@ -19,6 +19,10 @@ typeo = {git = "https://github.com/ML4GW/typeo.git", branch = "main"}
 
 "bbhnet.analysis" = {path = "../../../libs/analysis", develop = true}
 "bbhnet.logging" = {path = "../../../libs/logging", develop = true}
+bilby = "^1.4.1"
+
+[tool.poetry.group.dev.dependencies]
```

Comment:
Do we want to keep this dep or was this just for that notebook that's leftover?

Reply:
Actually wanted to ask about this. I added the `priors.py` from the `datagen` project into the vizapp project so we don't have to carry the `datagen` dep (which includes a `torch` dep). I guess the options are to make the `torch` dep an `extra` in the datagen project, or to refactor out `priors.py`.

Reply:
I think refactoring out `priors` definitely makes a lot of sense, especially since bilby can be managed by poetry. It an just be a one-file `bbhnet` library

Reply:
Okay. Maybe this should be done in #243 @wbenoit26

Reply:
If #243 looks good as-is, I'd rather get that merged and then do a quick second PR to refactor the priors

Reply:
I would agree that probably makes sense